projects/plugins/automattic-for-agencies-client/src/js/components/automattic-icon-logo/index.jsx

```diff
@@ -0,0 +1,37 @@
+import { Path, SVG } from '@wordpress/components';
+import React from 'react';
+
+/**
+ * Protect Logo component
+ *
+ * @param {object} props             - Component props
+ * @param {string} props.innerColor  - Color code for the line in the middle of the logo.
+ * @param {string} props.outerColor  - Color code for the logo's outer
+ * @returns {React.ReactElement} Component template
+ */
+export default function AutomatticIconLogo( { innerColor = '#00A3E0', outerColor = '#FFFFFF' } ) {
```

Comment:
Would it be worth adding those logos to the `@automattic/jetpack-components` package, with the other icons in there?
https://github.com/Automattic/jetpack/tree/231c92254c04e094d527b7eea670c957bb783889/projects/js-packages/components/components

Reply:
Resolved via https://github.com/Automattic/jetpack/pull/36664/commits/9b7f4b9b34d60514310a0e892564966161d38f17 :+1: